CHANGELOG.md

```diff
@@ -44,6 +44,7 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 ### Improvements
 
+* (simapp) [\#3874](https://github.com/cosmos/ibc-go/pull/3874) Refactor simapp to more closely resemble the style used in cosmos-sdk
```

Comment:
If you want you can remove this as the PR title, the squashed commit msg can be parsed by git cliff for changelogs (we're still experimenting with this tooling). 

You could rename the PR title as: `imp: refactor simapp to more closely resemble the style used in cosmos-sdk`

Reply:
ahhhhh thanks for explaining this, I understand where you're heading with commit messages and pr titles now